src/components/Main.jsx

```diff
@@ -0,0 +1,71 @@
+import style from './Main.module.css';
+import Person from '../assets/illustration-working.svg?react';
+import Brand from '../assets/icon-brand-recognition.svg?react';
+import Detailed from '../assets/icon-detailed-records.svg?react';
+import Fully from '../assets/icon-fully-customizable.svg?react';
+import { Button } from './Button';
+import { Text } from './Text';
+import { Title } from './Title';
+
+function Main() {
+  return (
+    <div className={style.main}>
+      <div className={style.more_img}>
+        <section className={style.more}>
+          <Title size={'normal'}>More than just shorter links</Title>
```

Comment:
I still cannot see any widget component. It's too many details for the Main component here.